settings/provisioner/topology/loadbalancer.yaml

```diff
@@ -0,0 +1,53 @@
+name: "loadbalancer"
+cpu: "2"
+memory: "4096"
+type: "external"
+os:
+    type: "linux"
+    variant: "rhel7"
+disks:
+    disk1:
+        path: "/var/lib/libvirt/images"
+        dev: "/dev/vda"
+        size: "20G"
+network:
+    provision_interface: "eth0"
+    oc_interface: "eth1"
+    ex_interface: "eth2"
+    undercloud:
+        oc_interface: "eth1"
+        internal_ip: 172.17.1.248/24
+    plain:
```

Comment:
same, these are osp-d related variables, please move it to settings/installer/ospd/<somewhere>